### PR TITLE
Completed tests for week 1 exercises

### PR DESCRIPTION
**Pull request by jenneft**
ready for review @cbanavi

### CODE REVIEW

File: src/JenTasks/week1/T2_JN_Division_Without_Operator.java
Comment (cbanavi) [inline src/JenTasks/week1/T2_JN_Division_Without_Operator.java:1]: Please don't use '/' in your code.

File: src/JenTasks/week1/T1_JN_Odd_Or_Even.java
Comment (cbanavi) [inline src/JenTasks/week1/T1_JN_Odd_Or_Even.java:1]: lgtm!

File: src/JenTasks/week1/T3_JN_FINRA.java
Comment (cbanavi) [inline src/JenTasks/week1/T3_JN_FINRA.java:1]: lgtm!

**Review by cbanavi — CHANGES_REQUESTED** [review]
T2 needs a revision

File: src/JenTasks/week1/T2_JN_Division_Without_Operator.java
Comment (jenneft) [reply src/JenTasks/week1/T2_JN_Division_Without_Operator.java:1]: On it.

**Comment by jenneft** [comment]
Changes completed

File: src/JenTasks/week1/T2_JN_Division_Without_Operator.java
Comment (cbanavi) [reply src/JenTasks/week1/T2_JN_Division_Without_Operator.java:1]: lgtm!

**Review by cbanavi — APPROVED** [review]
lgtm!

**Pull request merged**